### PR TITLE
Pivotal: ID: 175839554 : include user tag as part of extended submission tags

### PR DESCRIPTION
**Pull request by Juan-EBI**
https://www.pivotaltracker.com/story/show/175839554

### CODE REVIEW

File: commons/commons-model-extended-serialization/src/test/kotlin/uk/ac/ebi/extended/serialization/serializers/ExtFileSerializerTest.kt [hunk 1]
```diff
@@ -22,12 +22,12 @@ class ExtFileSerializerTest(private val tempFolder: TemporaryFolder) {
         val file = tempFolder.createFile("test-file.txt", "content")
         val extFile = ExtFile(
             file = file,
-            fileName = "test/path/test-file.txt",
```
Comment (jhoanmanuelms) [inline commons/commons-model-extended-serialization/src/test/kotlin/uk/ac/ebi/extended/serialization/serializers/ExtFileSerializerTest.kt:25]: I included this to cover the scenario of a file with an inner folder, I think it's worth having it.

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt [hunk 2]
```diff
@@ -28,9 +26,8 @@ class SecurityMapper {
 
     private fun getAllow(user: SecurityUser): List<String> {
         val accessTags = ArrayList<String>()
-        accessTags.add("~" + user.email)
-        accessTags.add("#" + user.id)
```
Comment (jhoanmanuelms) [inline submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt:32]: Are you sure we don't need this anymore?

**Review by jhoanmanuelms — APPROVED** [review]
LGTM. Please consider my comments.

File: submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt (hunk 2 above)
Comment (Juan-EBI) [reply submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt:32]: no we are not

Comment (awaisathar) [reply submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt:32]: Feel free to delete the "~" and the user id (including #). The front end only needs to match the accessTags in the submission with the allow list and already ignores # and ~.

**Pull request merged**